test_requirements/base.txt

```diff
@@ -1,7 +1,23 @@
 aldryn-apphook-reload>=0.2.1
+aldryn-apphooks-config
+aldryn-boilerplates
+aldryn-common>=0.1.3
+aldryn-reversion>=0.0.2
+aldryn-translation-tools
 coverage
+django-bootstrap3
+django-cms
```

Comment:
so why is this the case here https://github.com/aldryn/aldryn-newsblog/blob/master/test_requirements/base.txt @Marketionist ?